### PR TITLE
fix pwa navigation on localhost IP address

### PR DESCRIPTION
**Pull request by TrebuhD**

### CODE REVIEW

File: client/service-worker.js [hunk 1]
```diff
@@ -33,6 +33,6 @@ sw.router.registerRoute(
     sw.strategies.cacheFirst()
 );
 
-if (self.location.hostname !== 'localhost') {
+if (!self.location.hostname.match(/127.0.0.1|localhost/)) {
```
Comment (SamaneriVimala) [inline client/service-worker.js]: My local environment has gone to 172.18.0.1 for some reason.

Comment (TrebuhD) [reply client/service-worker.js]: That's weird. That seems to be an internal docker address. How did you discover it was the right IP for the site?

Comment (SamaneriVimala) [reply client/service-worker.js]: It says in the terminal what address it's using.

Comment (TrebuhD) [reply client/service-worker.js]: Ah! Yeah, that's obvious. I pushed the change, it should be OK now.

**Pull request merged**